vickev/howdypix
# Authentication

## Description

Introduce the Authentication flow with JWT and magic link!

What's not included in this PR:
 - Error page when the magic link is not valid anymore

## How to test?

- Check-out this branch.
- Go to the homepage and log in with `dev@vickev.com`

## Checklist
